### ISSUE TITLE
Remove the use of Identifying numbers in xAzurePack and xAzurePackUpdate

### ISSUE THREAD

**Issue by DdenBraver** [PowerShell/xAzurePack#8]
Currently xAzurePack and xAzurePackUpdate rest on the use of Identifying numbers to see if a producmt is installed.
This causes the need for the resource to be updated after every Update Rollup that is released.

Perhaps we can change this by checking the products using the Product Names instead of Identifying numbers.
Example:
Name              : Windows Azure Pack - Configuration Site - 2013
IdentifyingNumber : {282F6FAA-E6F1-4910-8A64-F20453944ACA}

We could use "Windows Azure Pack - Configuration Site - 2013" instead of {282F6FAA-E6F1-4910-8A64-F20453944ACA}

I could start working on this change, but I would like to hear opinions before I start working on this.

**Comment by DdenBraver** [comment]
PR #11 resolves this issue.

**Comment by J0F3** [comment]
Also get rid of the Win32_Product WMI Query -> https://support.microsoft.com/en-nz/kb/974524

**Comment by DdenBraver** [comment]
@J0F3 The class provided in the link is not available by default on a windows image.
Instead I will use the system registry to query for the installations.

```
(Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\*,
 HKLM:\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\* | Where-Object {![string]::IsNullOrWhiteSpace($_.DisplayName)} | Sort-Object DisplayName -Unique).DisplayName
```

**Comment by J0F3** [comment]
Yes, the resolution of the KB article is not really useful. I my opinion registry queries are the best options

**Comment by johlju** [comment]
The PR #11 has been labeled abandoned so that someone else in the community can continue the work and close this issue.
